protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -74,16 +74,22 @@ message DrillPBError{
      */
     RESOURCE = 7;
     /* equivalent to SQLNonTransientException.
+     * - out of memory
```

Comment:
for out of memory we throw "memoryError" which internally set's the errorType as RESOURCE not SYSTEM.